hack/bazel-test.sh

```diff
@@ -4,7 +4,7 @@ source hack/common.sh
 source hack/bootstrap.sh
 source hack/config.sh
 
-WHAT=${WHAT:-"//staging/src/kubevirt.io/client-go/... //pkg/... //cmd/... //tests/framework/..."}
```

Comment:
Since client-go also has test files, I have added the common path instead of having 2 paths, is there any reason for making the path explicit. (I mean what's the reasoning for preventing future additions from testing

Reply:
I wanted more visibility into these dedicated compatibility tests. This is subjective, no blocking